HW4/task3.py

```diff
@@ -0,0 +1,14 @@
+# Напишите функцию принимающую на вход только ключевые параметры и возвращающую словарь,
+# где ключ — значение переданного аргумента, а значение — имя аргумента.
+# Если ключ не хешируем, используйте его строковое представление.
+#
+
+def get_dict(first=None, second=None, third=None):
+    result = {}
+    for k, v in locals().items():
```

Comment:
Или можно было определить функцию, принимающую на вход, переменное количество именованных аргументов.
```suggestion
def get_dict(**kwargs):
    result = {}
    for k, v in kwargs.items():
```